edit-post/components/plugin-sidebar/error-boundary.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+
+class SidebarErrorBoundary extends Component {
```

Comment:
It should be called `PluginSidebarErrorBoundary` and the `className` used below should looks as follows: `plugin-sidebar__error-boundary`. See the guidelines: https://github.com/WordPress/gutenberg/blob/master/docs/coding-guidelines.md#naming